climpred/prediction.py

```diff
@@ -38,19 +47,62 @@ def compute_perfect_model(
         metric (str): `metric` name, see :py:func:`climpred.utils.get_metric_function`.
         comparison (str): `comparison` name, see
                           :py:func:`climpred.utils.get_comparison_function`.
+        dim (str or list): dimension to apply metric over. default: ['member', 'init']
         add_attrs (bool): write climpred compute args to attrs. default: True
 
     Returns:
-        skill (xarray object): skill score.
+        skill (xarray object): skill score with dimension as input without dim.
```

Comment:
I'm not sure what this means

Reply:
the dimensions of the input get reduced by `dim` into the output. so if `dim='init'` input has `lead, member, init`, output has `lead, member`